.gitmodules

```diff
@@ -94,3 +94,6 @@
 	path = tools/ipxe
 	url = https://github.com/ipxe/ipxe.git
 	branch = master
+[submodule "ec"]
+	path = ec
+	url = https://github.com/system76/ec.git
```

Comment:
Add branch = master to this, and update the submodule from master now that the VERSION override PR is merged